pygac/gac_io.py

```diff
@@ -77,10 +89,17 @@ def save_gac(satellite_name,
              mask, qual_flags, start_line, end_line, tsmcorr,
              gac_file, midnight_scanline, miss_lines, switch=None):
 
- 
+    along_track = lats.shape[0]
+    last_scan_line_number = qual_flags[-1, 0]
```

Comment:
As far as I remember, the last_scan_line_number attribute was included to enable the user to reconstruct the initial scanline number range. But if it is determined after slicing [temp_start_line:temp_end_line], scanline numbers >  temp_end_line are not taken into account. That's why I moved the line to the top of the method. If you think, it should just have the value of the last scanline number in the hdf5 file, we can move it to the end of the method.